elections.md

```diff
@@ -89,8 +89,8 @@ are announced early October, the steering committee should consider the
 following schedule:
 
 - election officers selected by July 1
-- voter eligibility criteria chosen by Aug 1
-- results announced at first community meeting after Oct 1
+- voter eligibility criteria chosen by Aug 23
```

Comment:
The dates need to be updated here also

21 August - Announce Election - first thing in the morning EST, this is
just sending an email
11 Sept - All bios and voting exemptions due - 5pm PST
18 Sept - Voting begins - "Morning of" to account for CIVS setup 
2 October - Voting ends - 5pm PST 
3 October - Results published on blog/community meeting - Normal
community meeting time, blog published right after.

Reply:
In hindsight from looking at this paragraph I was thinking it might be better to remove the dates from this steering doc entirely and just list these as goals that need to have date sets, that way this document can remain more generic and the specific dates for each election can be specified on their own document, that way we only have to make minor updates to the steering document. I'll push a PR update so we can see what that looks like.

Reply:
Good idea.